src/Symfony/Component/Validator/Constraints/Email.php

```diff
@@ -31,8 +31,12 @@ class Email extends Constraint
         self::HOST_CHECK_FAILED_ERROR => 'HOST_CHECK_FAILED_ERROR',
     );
 
+    const VALIDATION_MODE_HTML5 = 'html5';
+    const VALIDATION_MODE_RFC = 'rfc';
+    const VALIDATION_MODE_LOOSE = 'loose';
+
     public $message = 'This value is not a valid email address.';
     public $checkMX = false;
     public $checkHost = false;
-    public $strict;
+    public $mode = 'html5';
```

Comment:
This is a BC break for 3 reasons:

- the `strict` property does not work anymore
- the default behavior has changed)
- the constraint now has a default mode, meaning it is not possible to configure a global default anymore at the config level (as the validator uses it only when the constraint does not have a config)